libraries/adb/src/server/transport.ts

```diff
@@ -0,0 +1,77 @@
+import { PromiseResolver } from "@yume-chan/async";
+import { AbortController } from "@yume-chan/stream-extra";
+import type { ValueOrPromise } from "@yume-chan/struct";
+
+import type {
+    AdbIncomingSocketHandler,
+    AdbSocket,
+    AdbTransport,
+} from "../adb.js";
+import type { AdbBanner } from "../banner.js";
+
+import type { AdbServerClient } from "./client.js";
+
+export class AdbServerTransport implements AdbTransport {
```

Comment:
Add AdbServerTransport for "any device", "any USB device" and "any TCP device" to simulate ADB parameters.